eZ/Publish/Core/MVC/Symfony/View/BlockValueView.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\MVC\Symfony\View;
+
+use eZ\Publish\Core\FieldType\Page\Parts\Block;
+
+interface BlockValueView
```

Comment:
naming?

ref:
BlockView implementing BlockValueView, which sounds like two separate concepts, and should this interface extend View?

No good suggestions, just a hunch.